packages/wonder-blocks-tooltip/src/components/__tests__/tooltip-tail.test.js

```diff
@@ -26,92 +28,12 @@ describe("TooltipTail", () => {
             const makeNode = (p) => <TooltipTail placement={p} />;
 
             // Act
-            const testees = testPoints.map((tp) => () => shallow(makeNode(tp)));
+            const testees = testPoints.map((tp) => () => render(makeNode(tp)));
 
             // Assert
             for (const testee of testees) {
                 expect(testee).not.toThrowError();
             }
         });
     });
-
-    describe("INTERNALS", () => {
```

Comment:
**note:** I removed all the following tests b/c I noticed that these introduced false positives in the context of our current testing strategy. 

`_minDistanceFromCorners` was still being used in the code b/c it was being tested here, but it was not being used directly within the code implementation.

Instead of testing these methods, we can rely on FlowJS (or TS in the future) to ensure that we won't reach the impossible states that we are testing here.

Reply:
Using Chromatic for these tests cases makes sense. 👍